canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -285,28 +287,28 @@ $toolbar-divider-width: 1px;
 
 	.toolbar-text-content {
 		position: absolute;
-		width: 11px;
-		top: 15px;
-		left: 13px;
+		width: $spacing-04;
+		top: $spacing-05;
+		left: $spacing-04;
```

Comment:
The text is still off-centered. We'll have to hard code left back to `13px`
<img width="751" alt="image" src="https://github.com/user-attachments/assets/e959a8bd-3740-40e3-af7c-535ceed3be81">